docker/docker-entrypoint.sh

```diff
@@ -9,6 +9,34 @@ if [[ -n "${ENV_PATH}" ]]; then
   set +a
 fi
 
+# Bootstrap node from URL
+if [[ -n "${BOOTSTRAP_URL}" ]]; then
+  BOOTSTRAP_URL="${BOOTSTRAP_URL}/api/codex/v1/spr"
+  WAIT=${BOOTSTRAP_URL_WAIT:-300}
+  SECONDS=0
+  SLEEP=5
```

Comment:
```suggestion
  SLEEP=1
```
Maybe a shorter sleep?

Reply:
Updated